database/database.go

```diff
@@ -53,8 +57,27 @@ func Connect(cfg *config.KasparovFlags) error {
 	if err != nil {
 		return err
 	}
-
 	db.SetLogger(gormLogger{})
+
+	return validateTimeZone(db)
+}
+
+func validateTimeZone(db *gorm.DB) error {
+	result := struct {
+		GlobalTimeZone, SystemTimeZone string
+	}{}
+	dbResult := db.Raw("SELECT @@global.time_zone as global_time_zone, @@global.system_time_zone as system_time_zone").Scan(&result)
+	dbErrors := dbResult.GetErrors()
+	if httpserverutils.HasDBError(dbErrors) {
+		return httpserverutils.NewErrorFromDBErrors("some errors were encountered when checking the database timezone:", dbErrors)
+	}
+	timeZone := result.GlobalTimeZone
+	if result.GlobalTimeZone == globalTimeZoneSystem {
+		timeZone = result.SystemTimeZone
+	}
+	if timeZone != expectedTimeZone {
+		return errors.Errorf("timezone is %s but is expected to be %s", timeZone, expectedTimeZone)
```

Comment:
Please write a more informative error message, a long the lines of "To prevent conversion errors - Kasparov should only run with a database configured to use the UTC timezone, currently configured timezone is blablabla"